src/librustc_session/filesearch.rs

```diff
@@ -143,7 +143,7 @@ fn find_libdir(sysroot: &Path) -> Cow<'static, str> {
     // FIXME: This is a quick hack to make the rustc binary able to locate
     // Rust libraries in Linux environments where libraries might be installed
     // to lib64/lib32. This would be more foolproof by basing the sysroot off
-    // of the directory where librustc is located, rather than where the rustc
+    // of the directory where librustc_middle is located, rather than where the rustc
```

Comment:
This should probably say `librustc_driver`, if it refers to the dynamic library (since it's the only one left).